css-three-column-layout/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  background: #1896d9;
+  font-family: Lato, sans-serif;
+  color: white;
+}
+
+.container {
+  max-width: 1100px;
+  margin: 30px auto 0;
+}
+
+.row {
+  display: flex;
+  justify-content: space-around;
+}
+
+.column-one-third {
+  flex-basis: calc((100% / 3) - 5%);
+}
+
+.column-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.img-container {
+  background: white;
+  width: 140px;
+  height: 140px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  margin: 15px 0;
+}
+
+.img-container > div {
+  flex-basis: 50%;
+  height: 50%;
+}
+
+img {
+  width: 100%;
+  object-fit: cover;
+}
```

Comment:
Instead of having a `div` that is 50% of the container and an `img` that is 100% of the 50%, can you remove the `div` and just set the `img` to have a width/flex-basis of 50%?